### PR TITLE
libvirt_cgroup: fix cpusest.cpus missing

### PR DESCRIPTION
**Pull request by dzhengfy**
'sys/fs/cgroup/cpuset/machine.slice/cpuset.cpus' file could be created
when the vm is started for the first time. This is to make sure the file
exist in the test environment.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
@liuyd96

File: virttest/libvirt_cgroup.py [hunk 1]
```diff
@@ -470,7 +470,7 @@ def set_cpuset_cpus(self, value):
         cmd = "echo %s > /sys/fs/cgroup/cpuset/machine.slice/cpuset.cpus" % value
         process.run(cmd, ignore_status=False, shell=True)
 
-    def get_cpuset_cpus(self):
+    def get_cpuset_cpus(self, vm_name):
```
Comment (Yingshun) [inline virttest/libvirt_cgroup.py:473]: Please add docstring for 'vm_name'

Comment (liuyd96) [inline virttest/libvirt_cgroup.py:473]: I think set_cpuset_cpus also need a check for "/sys/fs/cgroup/cpuset/machine.slice/cpuset.cpus"
others LGTM. Thanks.

Comment (dzhengfy) [reply virttest/libvirt_cgroup.py:494]: updated

Comment (dzhengfy) [reply virttest/libvirt_cgroup.py:494]: updated

**Review by liuyd96 — APPROVED** [review]

**Review by Yingshun — APPROVED** [review]

**Pull request merged**